x-pack/test/functional_synthetics/apps/uptime/synthetics_integration.ts

```diff
@@ -73,6 +73,7 @@ export default function (providerContext: FtrProviderContext) {
             },
           },
         ],
+        origin: 'ui',
```

Comment:
This doesn't seem like it should be here. These tests are for Elastic Synthetics legacy policies, not for private locations. Are we applying this key on everything?